math/area.cpp

```diff
@@ -76,6 +76,20 @@ T parallelogram_area(T base, T height) {
     return base * height;
 }
 
+ /**
+ * @brief area of a
+ * [trapezoid/trapezium with two parallel
+ * side](https://en.wikipedia.org/wiki/Trapezoid) ((b1 + b2)/2]*h)
+ * @param base1 is the length of one of the parallel side of the trapezium
```

Comment:
You may want to describe the `T` param as well.
```suggestion
 * @tparam <description>
 * @param base1 is the length of one of the parallel side of the trapezium
```